lib/samwise/client.rb

```diff
@@ -32,6 +40,18 @@ def excluded?(duns: nil)
 
     def small_business?(duns: nil)
       response = lookup_duns(duns: duns)
+      parse_response_for_small_business(response)
+    end
+
+    private
+
+    def parse_response_for_sam_status(response)
+      response.status == 200
+    end
+
+    def parse_response_for_small_business(response)
+      return false if response.code == 404
```

Comment:
Should this really be false if any code besides 200 (and also the stuff below)? Although I guess in those cases it would fail when trying to parse the JSON